javascript/javascript1/week4/sentiment.js

```diff
@@ -0,0 +1,59 @@
+function getWordsFromText (text, wordsList) {
+   const words = [];
+   for (let i = 0; i < text.length; i++ ){
+      if(wordsList.includes(text[i])) {
```

Comment:
This won't work if case is different. For example if in `text` you have `brilliant` and in the `arrPositiveWords` you have it with capital letter (`Brilliant`). Using `toLowerCase()` can prevent such problem.